docs/en/connector-v2/sink/Kingbase.md

```diff
@@ -0,0 +1,159 @@
+# Kingbase
+
+> JDBC Kingbase Sink Connector
+
+## Support Those Engines
+
+> Spark<br/>
+> Flink<br/>
+> SeaTunnel Zeta<br/>
+
+## Key Features
+
+- [x] [exactly-once](../../concept/connector-v2-features.md)
+- [ ] [cdc](../../concept/connector-v2-features.md)
+
+## Description
+
+> Use `Xa transactions` to ensure `exactly-once`. So only support `exactly-once` for the database which is
+> support `Xa transactions`. You can set `is_exactly_once=true` to enable it.
```

Comment:
@ic4y Does kingbase have this feature?

Reply:
i will test the feature

Reply:
> @ic4y Does kingbase have this feature?

sorry kingbase don't support the feature

Reply:
`com.kingbase8.util.KSQLException: ERROR: prepared transactions are disabled
  Hint: Set max_prepared_transactions to a nonzero value.`
need Set max_prepared_transactions to a nonzero value

Reply:
<img width="1313" alt="image" src="https://github.com/apache/seatunnel/assets/72905543/8a8367e9-e05b-42ac-9ad8-fa96b2f9e9f8">
i try it but not ok

Reply:
don't support